src/coreclr/debug/createdump/crashinfo.cpp

```diff
@@ -427,6 +420,100 @@ uint64_t CrashInfo::GetBaseAddress(uint64_t ip)
     return found->Offset();
 }
 
+//
+// Returns the module base address for the given module name or 0 if not found.
+//
+uint64_t
+CrashInfo::GetBaseAddress(const char* moduleName)
+{
+    for (const ModuleInfo& moduleInfo : m_moduleInfos)
+    {
+        std::string name = GetFileName(moduleInfo.ModuleName());
+        if (strcmp(name.c_str(), moduleName) == 0)
```

Comment:
I made that change for Linux, but for MacOS we need to use strcasecmp because MacOS file paths are case insensitive. I must have forgot or it never clicked until Lee pointed that out.  I think we have code in the native and managed SOS function that also will eventually need to be fixed.